### ISSUE TITLE
Hand range in creative mode is wrong

### ISSUE THREAD

**Issue by GreenXenith** [minetest-mods/skinsdb#92]
Hand range should be higher (10? I dont remember) when creative mode is enabled.

**Comment by SmallJoker** [comment]
This is actually a Minetest limitation. The hand range is overwritten by the "hand" slot item: https://github.com/minetest/minetest/blob/b1dec37adb16f7ccd21209cbdd8381865cdfb5ca/src/client/game.cpp#L3241-L3245

**Comment by GreenXenith** [comment]
> This is actually a Minetest limitation. The hand range is overwritten by the "hand" slot item: https://github.com/minetest/minetest/blob/b1dec37adb16f7ccd21209cbdd8381865cdfb5ca/src/client/game.cpp#L3241-L3245

That is correct, but the range of the custom hand item can be changed if creative mode is enabled when registered.
